test/test_repo_template/asv_test_repo/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 from __future__ import absolute_import, division, unicode_literals, print_function
 
-dummy_value = {dummy_value}
+dummy_value = {dummy_value}  # noqa:F821 dummy_value not defined but used
```

Comment:
Can you create an issue for this, and link it here? It may be worth running this test file to see why this test file is not failing in the CI.